location/src/commonMain/kotlin/location/LocationManager.kt

```diff
@@ -50,12 +51,10 @@ abstract class BaseLocationManager(
         if (monitoringPermissionsJob.compareAndSet(null, job))
             launch(job) {
                 locationPermissionRepo.collect { state ->
-                    when (state) {
-                        is PermissionState.Denied.Requestable -> if (autoRequestPermission) state.request(permissions.getManager(locationPermission))
-                        else -> {
-                        }
-                    }
+                    if (state is PermissionState.Denied.Requestable&& autoRequestPermission)
```

Comment:
Missing space between `PermissionState.Denied.Requestable` and `&&`. Doesn't it cause errors?